StandardLibrary/runtimes/python/src/smithy_dafny_standard_library/internaldafny/extern/Time.py

```diff
@@ -8,10 +8,10 @@
 # Extend generated class with our externs
 class default__(smithy_dafny_standard_library.internaldafny.generated.Time.default__):
     def CurrentRelativeTimeMilli():
-        return datetime.datetime.now(tz = pytz.UTC).timestamp() * 1000
+        return round(datetime.datetime.now(tz = pytz.UTC).timestamp() * 1000)
```

Comment:
Should we round or truncate the milliseconds?

It looks like Java truncates but .NET rounds.

It probably doesn't matter either way